### PR TITLE
Add config to support spark2.2

### PR DESCRIPTION
**Pull request by heyu1**
Signed-off-by: He, Yu <yu.he@intel.com>

### CODE REVIEW

**Comment by heyu1** [comment]
@carsonwang Please help to merge, thanks!

File: docs/build-hibench.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Because some Maven plugins cannot support Scala version perfectly, there are som
 
 
 ### Specify Spark Version ###
-To specify the spark version, use -Dspark=xxx(1.6, 2.0 or 2.1). By default, it builds for spark 2.0
+To specify the spark version, use -Dspark=xxx(1.6, 2.0, 2.1 or 2.2). By default, it builds for spark 2.0
```
Comment (gczsjdy) [inline docs/build-hibench.md]: Why don't we use Spark 2.2 as default?

File: .travis.yml [hunk 2]
```diff
@@ -30,6 +30,7 @@ cache:
   directories:
   - $HOME/.m2
 script:
+  - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=2.2 -Dscala=2.11
   - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=2.0 -Dscala=2.11
```
Comment (gczsjdy) [inline .travis.yml]: The original is lack of script for Spark 2.1? @carsonwang

Comment (carsonwang) [reply .travis.yml]: Yes. There is a timeout limit on travis so we didn't build for all profiles.

File: docs/build-hibench.md (hunk 1 above)
Comment (carsonwang) [reply docs/build-hibench.md]: We may not want to update this to the latest version and user is always encouraged to specify the spark version when building HiBench. We can update this when we release the next HiBench version.

**Comment by carsonwang** [comment]
Thanks all! I'll merge this.

**Pull request merged**